app/utils/CheckYourAnswersHelper.scala

```diff
@@ -60,8 +60,8 @@ class CheckYourAnswersHelper(userAnswers: UserAnswers, countryOptions: CountryOp
 
   def addressAnswer(address: Address): Seq[String] = {
     val country = countryOptions.options.find(_.value == address.country).map(_.label).getOrElse(address.country)
-    Seq(Some(s"${address.addressLine1},"), Some(s"${address.addressLine2},"), address.addressLine3.map(line3 => s"$line3,"),
-      address.addressLine4.map(line4 => s"$line4,"), address.postcode.map(postCode => s"$postCode,"), Some(country)).flatten
+    Seq(Some(address.addressLine1), Some(address.addressLine2), address.addressLine3,
```

Comment:
would be nice to have some unit test for this, its seems its missing.

Reply:
Done